pkg/classroom/classroom.go

```diff
@@ -100,6 +100,14 @@ type GitHubOrganization struct {
 }
 
 func NewAssignmentList(assignments []Assignment) AssignmentList {
+	if len(assignments) == 0 {
+		return AssignmentList{
```

Comment:
My only Nit is I'd like to see these wrapped behind a `NewEmptyAssignmentList` function. @jalafel But it's fine.

Reply:
I initially had done that but it does extend the types across the codebase a bit, so I will save this for a different PR!